src/server/upload/helpers/upload-validation.js

```diff
@@ -4,4 +4,11 @@ const uploadPathValidation = Joi.object({
   id: Joi.string().uuid().required()
 })
 
-export { uploadPathValidation }
+const validateFilePayload = Joi.object({
```

Comment:
i dont think we need payload validation. the tenant service defines what's in the form and can validate the content on their /redirect handler

Reply:
Note this is not the typical payload validation that happens in `route.options.validate.payload`. This validation, validates the payload in the handler and adds the error to Redis, so this error can then be obtained via the status endpoint on the clients redirect error page.

I was thinking the basics on a file input such as no file, to large, incorrect mime type, could be supplied in this way and then for custom work, as you said:

> the tenant service defines what's in the form and can validate the content on their /redirect handler

But if we want to push all validation to the tenant I can work with  that. Maybe a catch up after SU?

Reply:
Re: discussion thats the error stuff now removed